woocommerce/woocommerce-android
# Run onboarding check on the ipp hub screen in async way

Closes: #7024


### Description

The PR adds onboarding check to be done in an async manner on the hub screen


### Testing instructions

* Try the case when onboarding is complete - all (4 or 5 if Stripe and WCpay are active) buttons should be active
* Try the case when onboarding fails - all buttons active except manage card readers

### Images/gif


https://user-images.githubusercontent.com/4923871/182571787-6d823f6e-fc21-4e14-8ae1-10d73e2dcf6f.mp4


https://user-images.githubusercontent.com/4923871/182572813-e306c483-af83-47fc-b55e-1a4d147d6028.mp4
